src/celeritas/CMakeLists.txt

```diff
@@ -140,7 +140,6 @@ if(CELERITAS_USE_ROOT)
   # Call the ROOT library generation inside a function to prevent ROOT-defined
   # directory properties from escaping
```

Comment:
Since this no longer applies, and the function is only called once (just for this purpose), can you delete the function and just inline the content?

Reply:
Done in latest commit - the use of `include_directories` is safe in the sense it's only adding project local paths, as is the setting of `CMAKE_LIBRARY_OUTPUT_DIRECTORY`.

Longer term, I _think_ use of `include_directories` can be removed completely if this code block is moved _after_ the declaration of the `celeritas` target. ROOT will then pick up usage requirements from the target. However, that will require a `celeritas_target_sources` command analogous to `target_sources` to add the object lib to `celeritas` afterwards. Best done as/when that functionality becomes needed more widely.